src/shogun/lib/SGMatrix.h

```diff
@@ -57,6 +57,25 @@ template<class T> class SGMatrix : public SGReferencedData
 		/** Constructor to create new matrix in memory */
 		SGMatrix(index_t nrows, index_t ncols, bool ref_counting=true);
 
+		/**
+		 * Constructor for creating a SGMatrix from a SGVector with refcounting.
```

Comment:
Even if it should be enough saying "with refcounting" as it is, I would perhaps remark here that no data is copied. Also in the method below.